his_checker.py

```diff
@@ -9,22 +9,35 @@
 
 import sys
 import argparse
+import os
 
-METRIC_FAIL = 0
+CYCLOMATIC_THRESHOLD = 10
 
-def process_metric(filenames):
+def process_complexity(filename):
```

Comment:
I would change the `filename` parameter to `files`

Reply:
It makes sense to change the name, thanks!